features/in_onderzoek.feature

```diff
@@ -166,188 +165,283 @@ Functionaliteit: in onderzoek
+      | naam                | waarde    |
+      | burgerservicenummer | 555550001 |
+      En de persoon heeft de volgende 'verblijfplaats' gegevens
+      | naam                                     | waarde   |
+      | straatnaam (11.10)                       | straat   |
+      | aanduiding gegevens in onderzoek (83.10) | <waarde> |
+      | datum ingang onderzoek (83.20)           | 20220307 |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                              |
+      | type                | RaadpleegMetBurgerservicenummer     |
+      | burgerservicenummer | 555550001                           |
+      | fields              | verblijfplaats.verblijfadres.<veld> |
+      Dan heeft de response een persoon met de volgende 'verblijfplaats' gegevens
+      | naam                                                       | waarde       |
+      | type                                                       | Adres        |
+      | verblijfadres.inOnderzoek.<veld>                           | true         |
+      | verblijfadres.inOnderzoek.datumIngangOnderzoek.datum       | 2022-03-07   |
+      | verblijfadres.inOnderzoek.datumIngangOnderzoek.type        | Datum        |
+      | verblijfadres.inOnderzoek.datumIngangOnderzoek.langFormaat | 7 maart 2022 |
+
+      Voorbeelden:
+      | waarde | veld                    |
+      | 081120 | huisnummer              |
+      | 081130 | huisletter              |
+      | 081140 | huisnummertoevoeging    |
+      | 081150 | aanduidingBijHuisnummer |
+      | 081160 | postcode                |
+      | 081170 | woonplaats              |
+
+    Abstract Scenario: persoon heeft <veld> van verblijfplaats in onderzoek
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 555550001 |
+      En de persoon heeft de volgende 'verblijfplaats' gegevens
+      | naam                                     | waarde              |
+      | locatiebeschrijving (12.10)              | locatiebeschrijving |
+      | aanduiding gegevens in onderzoek (83.10) | <waarde>            |
+      | datum ingang onderzoek (83.20)           | 20220307            |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                              |
+      | type                | RaadpleegMetBurgerservicenummer     |
+      | burgerservicenummer | 555550001                           |
+      | fields              | verblijfplaats.verblijfadres.<veld> |
+      Dan heeft de response een persoon met de volgende 'verblijfplaats' gegevens
+      | naam                                                       | waarde              |
+      | type                                                       | Locatie             |
+      | verblijfadres.locatiebeschrijving                          | locatiebeschrijving |
```

Comment:
Deze regel kan:
* verwijderd worden (in het voorgaande scenario is deze ook niet aanwezig)
* of gewijzigd worden in:
   | verblijfadres.<veld> | <locatiebeschrijving> |

Bij de laatste optie moet er in de voorbeelden tabel ook een kolom 'locatiebeschrijving' worden toegevoegd.